README.md

```diff
@@ -18,61 +20,93 @@ Citation:
 
 ## Quickstart Guide
 
-### Installation
+1. Install PyAudio and and PyTorch through your distribution's package system.
+
+2. Install Howl using `pip`
+
+```
+pip install howl
+```
+
+3. To immediately use a pre-trained Howl model for inference, we provide the `client` API. The following example loads the "hey_fire_fox" pretrained model with a simple callback and starts the inference client.
+
+```
+from howl.client import HowlClient
```

Comment:
Here is the example for testing

Reply:
I think we should have a file for this along with this code in the Readme

Reply:
Oh you mean like a basic demo file?

Reply:
Added a file to examples/hey_fire_fox.py

Reply:
can you update the read me?